### PR TITLE
solve the interpolation issue

### PR DESCRIPTION
**Pull request by Kvieta1990**
#### Summary of work

To solve the problem with the `InterpolateBackground` algorithm. The previous implementation was wrong, giving nonsense result.

### To test:

```python
# import mantid algorithms, numpy and matplotlib
from mantid.simpleapi import *
import matplotlib.pyplot as plt
import numpy as np

# create workspace
dataX = [0,1,2,3,4,5,6,7,8,9] # or use dataX=range(0,10)
dataY1 = [1,1,1,1,1,1,1,1,1,1] # or use dataY=[1]*10
dataY2 = [2,2,2,2,2,2,2,2,2,2] # or use dataY=[2]*10
ws1 = CreateWorkspace(dataX, dataY1)
ws2 = CreateWorkspace(dataX, dataY2)
wsGroup = GroupWorkspaces("ws1,ws2")
interpoTemp = "200"
# CreateWorkspace does not add the "SampleTemp" property so we need to add it here
ws1.getRun().addProperty("SampleTemp", "100", False)
ws2.getRun().addProperty("SampleTemp", "300", False)

# Perform the background interpolation
outputWS = InterpolateBackground(wsGroup, interpoTemp)

# Check output
print("Interpolated Y values are: {}".format(outputWS.readY(0)))
```

For every point, we are expected a returned value of `1.5`, obviously.

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

File: Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/InterpolateBackground.py [hunk 1]
```diff
@@ -74,8 +74,9 @@ def PyExec(self):
         lowWS = mtd[self.low_ws]
         highWS = mtd[self.high_ws]
 
-        scalar = (self.interpo_temp - self.low_temp) / (self.high_temp - self.interpo_temp)
-        outputWS = lowWS + scalar * highWS - scalar * lowWS
```
Comment (AndreiSavici) [inline Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/InterpolateBackground.py:78]: Could be simplified to  `outputWS = lowWS + scalar * highWS +(1- scalar) * lowWS`

File: docs/source/algorithms/InterpolateBackground-v1.rst [hunk 2]
```diff
@@ -20,7 +20,9 @@ Below is the function used for the interpolation:
 
 .. math::
 
-	WS_{interpolated} = WS_{low} + (temp_{interpo} - temp_{low}) / (temp_{high} - temp_{interpo}) * (WS_{high} - WS_{low}),
+    factor1 = (temp_{interpo} - temp_{low}) / (temp_{high} - temp_{low})
```
Comment (AndreiSavici) [inline docs/source/algorithms/InterpolateBackground-v1.rst:23]: See my comment on the code

**Review by AndreiSavici — COMMENTED** [review]
I think you can simplify the code

File: Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/InterpolateBackground.py (hunk 1 above)
Comment (Kvieta1990) [reply Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/InterpolateBackground.py:78]: This is addressed with a new commit. Thank you!

File: docs/source/algorithms/InterpolateBackground-v1.rst (hunk 2 above)
Comment (Kvieta1990) [reply docs/source/algorithms/InterpolateBackground-v1.rst]: addressed. thanks

**Review by AndreiSavici — APPROVED** [review]

**Pull request merged**